src/modules/launcher/Plugins/Microsoft.Plugin.Program/Languages/pl.xaml

```diff
@@ -34,4 +34,9 @@
     <system:String x:Key="wox_plugin_program_plugin_name">Programy</system:String>
     <system:String x:Key="wox_plugin_program_plugin_description">Szukaj i uruchamiaj programy z poziomu Woxa</system:String>
 
+    <!--Application subtitle-->
```

Comment:
@crutkas .  Do you know if there is any action required on our part to get these localized?

Reply:
i think we should rip out the current set of loc files actually and have them go through our loc system.  for a first pass